products/cloudiot/terraform.yaml

```diff
@@ -0,0 +1,144 @@
+          cloudiot_additional_device_telemetry_topic_name: "additional-telemetr"
+          cloudiot_subfolder_matches_additional_device_telemetry_topic: "test/path"
+        test_env_vars:
+          project: :PROJECT_NAME
+          region: :REGION
+    properties:
+      id: !ruby/object:Overrides::Terraform::PropertyOverride
+        required: true
+        name: 'name'
+        description: |
+          A unique name for the resource, required by device registry.
+        validation: !ruby/object:Provider::Terraform::Validation
+          function: 'validateCloudIotDeviceRegistryID'
+      name: !ruby/object:Overrides::Terraform::PropertyOverride
+        # We don't need this field, because it has the same format as the ID
+        exclude: true
+      region: !ruby/object:Overrides::Terraform::PropertyOverride
+        ignore_read: true
+        required: false
+        default_from_api: true
+        description: |
+          The region in which the created registry should reside.
+          If it is not provided, the provider region is used.
+      logLevel: !ruby/object:Overrides::Terraform::PropertyOverride
+        name: 'log_level'
+        diff_suppress_func: 'emptyOrDefaultStringSuppress("")'
+      eventNotificationConfigs: !ruby/object:Overrides::Terraform::PropertyOverride
+        name: 'event_notification_configs'
+        default_from_api: true
+      eventNotificationConfigs.subfolderMatches: !ruby/object:Overrides::Terraform::PropertyOverride
+        name: 'subfolder_matches'
+        validation: !ruby/object:Provider::Terraform::Validation
+          function: 'validateCloudIotDeviceRegistrySubfolderMatch'
+      eventNotificationConfigs.pubsubTopicName: !ruby/object:Overrides::Terraform::PropertyOverride
+        name: 'pubsub_topic_name'
+        diff_suppress_func: 'compareSelfLinkOrResourceName'
+      stateNotificationConfig: !ruby/object:Overrides::Terraform::PropertyOverride
+        # Excluding this because the original (manually-generated) implementation
+        # wrongly set this to be a Map, instead of a NestedObject. To avoid breaking
+        # changes, we observe that behaviour by excluding this field and adding a
+        # corresponding custom element (of type Map) to the schema. When we're
+        # ready to introduce this breaking change, remove this "exclude" directive
+        # along with the corresponding custom schema element.
+        exclude: true
+        name: 'state_notification_config'
+      stateNotificationConfig.pubsubTopicName: !ruby/object:Overrides::Terraform::PropertyOverride
+        # See the comment on stateNotificationConfig.exclude
+        exclude: true
+        name: 'pubsub_topic_name'
+        diff_suppress_func: 'compareSelfLinkOrResourceName'
+      mqttConfig: !ruby/object:Overrides::Terraform::PropertyOverride
+        # See the comment on stateNotificationConfig.exclude
+        exclude: true
+        name: 'mqtt_config'
```

Comment:
Why are you changing the name on fields you're excluding? Also I don't think this line will do anything since we'll case it correctly in the generator.

Reply:
Ok!

Reply:
done